packages/ui/src/utils/button.ts

```diff
@@ -8,6 +8,7 @@ export const buttonBase = css`
   background: transparent;
   border: none;
   cursor: pointer;
+  padding: 0;
```

Comment:
Hadn't realized that buttons have built-in padding by default in Chrome, so I removed that in `buttonBase`.